.github/workflows/release.yml

```diff
@@ -27,6 +27,7 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           pypi-token: ${{ secrets.PYPI_TOKEN }}
           pre-tag: |
+            pip install setuptools  # needed for setup.py --version
```

Comment:
@yarikoptic This is for `setup.py --name`, not `setup.py --version` (which you're not doing).

Reply:
dang... right... damn suggestions in pycharm... will all be gone with hardcoding...